shacl12-core/index.html

```diff
@@ -1199,7 +1199,7 @@ <h3>Constraints, Parameters and Constraint Components</h3>
 					</p>
 					<p>
 						For a <a>constraint component</a> <code>C</code> with <a>mandatory parameters</a> <code>p1</code>, ... <code>pn</code>,
-						a <a>shape</a> <code>s</code> in a <a>shapes graph</a> <code>SG</code> <em>declares</em> a <dfn data-lt="constraints">constraint</dfn>
+						a <a>shape</a> <code>s</code> in a <a>shapes graph</a> <code>SG</code> <em>declares</em> a <dfn>constraint</dfn>
```

Comment:
As in my earlier comment.
```suggestion
						a <a>shape</a> <code>s</code> in a <a>shapes graph</a> <code>SG</code> <em>declares</em> a <dfn data-lt="constraint|constraints">constraint</dfn>
```